### PR TITLE
Adds a helper to run tasks

### PR DESCRIPTION
**Pull request by jaresty**
VSCode allows you to run tasks automatically. This makes that easily accessible to anyone running talon.

### CODE REVIEW

**Comment by jaresty** [comment]
This seems to be failing due to an error in the ci for some reason:

```
stderr:
        ERROR: Package 'talonfmt' requires a different Python: 3.12.2 not in '<3.12,>=3.7.1'
    Check the log at /pc/pre-commit.log
```

File: apps/vscode/vscode.talon [hunk 1]
```diff
@@ -247,6 +247,10 @@ terminal scroll up: user.vscode("workbench.action.terminal.scrollUp")
 terminal scroll down: user.vscode("workbench.action.terminal.scrollDown")
 terminal <number_small>: user.vscode_terminal(number_small)
 
+task run [<user.text>]:
+    user.vscode("workbench.action.quickOpen")
+    insert("task ")
```
Comment (pokey) [inline apps/vscode/vscode.talon:252]: I believe you can just do

```suggestion
    user.vscode("workbench.action.tasks.runTask")
```

Comment (jaresty) [reply apps/vscode/vscode.talon]: That works great-thank you!

**Review by pokey — APPROVED** [review]
LGTM!

**Pull request merged**